framework/0019-edtf-formatting.md

```diff
@@ -0,0 +1,261 @@
+- Start Date: 2020-03-24
+- RFC PR: [#19](https://github.com/inveniosoftware/rfcs/pull/19)
+- Authors: Lars Holm Nielsen
+- State: DRAFT
```

Comment:
Question: This states is `draft` until it has been implemented? or until it has been merged?